include/swift/ABI/Task.h

```diff
@@ -562,7 +562,7 @@ class AsyncTask : public Job {
   }
 
   /// Returns true if the current task has any task preference record,
-  /// including if it has an initial task preference record or onces
+  /// including if it has an initial task preference record or once
```

Comment:
I think this should actually be "one" (as in "or one set during the lifetime of the task").